packages/ERTP/CONTRIBUTING.md

```diff
@@ -4,44 +4,42 @@ Thank you!
 
 ## Contact
 
-We use github issues for all bug reports: https://github.com/Agoric/ERTP/issues
+We use github issues for all bug reports:
+https://github.com/Agoric/agoric-sdk/issues Please add an [ERTP]
+prefix to the title and ERTP tag to ERTP-related issues.
 
 ## Installing, Testing
 
 You'll need Node.js version 11 or higher. 
 
-* git clone https://github.com/Agoric/ERTP/
-* npm install
-* npm test
+* git clone https://github.com/Agoric/agoric-sdk/
+* cd packages/ERTP
+* yarn install
+* yarn test
```

Comment:
Not sure whether we should point folks at doing `yarn test` in the package (running only the ERTP tests), or running it from the top-level (where we get all the cross-package tests too). Might be ok to let CI do the cross-package tests, as long as PR submitters know it's not going to get merged if it breaks some other repo's tests.